# service type for image geometry projection
float64 x
float64 y
float64 z
---
float64 x
float64 y
